src/app/i18n/I18nProvider.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { I18nextProvider } from 'react-i18next';
```

Comment:
don't forget to add `i18next` and `react-i18next` to `package.json` via `yarn add -E ...` (let's keep exact versions to avoid update surprises)